src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Apprentice/Index.cshtml

```diff
@@ -344,7 +344,14 @@ else
                         @apprenticeship.CourseName
                     </td>
                     <td class="govuk-table__cell" data-label="Planned start date">
-                        @apprenticeship.PlannedStartDate.ToGdsFormatWithoutDay()
+                        @if (apprenticeship.IsOnFlexiPaymentPilot.GetValueOrDefault() && apprenticeship.ActualStartDate.HasValue)
```

Comment:
`(apprenticeship.IsOnFlexiPaymentPilot is true && apprenticeship.ActualStartDate.HasValue)`